frontend/apps/filemanager/filemanagercollection.lua

```diff
@@ -878,6 +942,9 @@ function FileManagerCollection:searchCollections()
         end
         return _files_found, _files_found_order
     end, info)
+    if is_cre_cache_disabled then
+        require("document/credocument").cacheInit()
+    end
```

Comment:
If I understand it right, here we are in the main process.
You will set `is_cre_cache_disabled=true` in `isFileMatch()`, which is run in the subprocess. So, the `is_cre_cache_disabled` in the main process will not be affected: the 2 processes are independant and don't share any variable/memory.
You don't even need to re-enable the cache: once the subprocess dies, it's gone, whatever the state of things is, we don't care. The main process never gets it disabled.